src/containers/AddEventPage.js

```diff
@@ -14,6 +16,13 @@ export class AddEventPage extends React.Component {
   }
 }
 
+AddEventPage.propTypes = {
+  addEvent: PropTypes.func.isRequired,
+  history: PropTypes.shape({
```

Comment:
You are using the same `history` config in prop types in multiple places. My suggestion is to create a new file with common prop types (name it as you want) and create there an object with that history prop type. Then, you would be able to import that to all your locations and do something like:
```
AddEventPage.propTypes = {
  addEvent: PropTypes.func.isRequired,
  ...historyPropTypes
```